config/jobs.txt

```diff
@@ -34,7 +34,7 @@ Medical Doctor=7,4
 Paramedic=2,1
 Chemist=3,2
 Geneticist=3,2
-Virologist=1,1
+Virologist=2,2
```

Comment:
Honestly I would make this 2,1

Reply:
I added new spawn positions as well

Reply:
It doesn't matter. I don't want two viros to spawn by default, but rather let another one join if they want to.

Reply:
that's... quite literally what this is for though. Latejoining places you at a disadvantage.